learnAndCodeFinalProject/src/main/java/org/learnAndCode/Server/Server.java

```diff
@@ -0,0 +1,22 @@
+package org.learnAndCode.Server;
+
+import java.io.*;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+
+public class Server {
+    public static void main(String[] args) {
+        try (ServerSocket serverSocket = new ServerSocket(9202)) {
```

Comment:
remove the hardcoded value 9202 values doesn't give clarity what it is used for. create a variable and use.